frontend/app/services/TouchpointBackend.scala

```diff
@@ -63,7 +63,7 @@ object TouchpointBackend {
     // extendedRunner sets the configurable read timeout, which is used for the createSubscription call.
     val extendedRunner = RequestRunners.configurableFutureRunner(20.seconds)
 
-    val zuoraSoapClient = new ClientWithFeatureSupplier(FeatureChoice.codes, config.zuoraSoap, runner, extendedRunner)
+    val zuoraSoapClient = new ClientWithFeatureSupplier(Set.empty, config.zuoraSoap, runner, extendedRunner)
```

Comment:
Legacy Patron Members could select either Books or Events benefits on checkout. These were stored as Features in the Zuora record. If they select Events, rather than Books, then the site will give them access to the discounted events.

Reply:
As long as that logic still works after this then the change is safe.

Reply:
This logic is implemented via https://github.com/guardian/membership-frontend/blob/master/frontend/app/model/FeatureChoice.scala so it is not using membership-common for it. When patrons want to renew, what is the process? Do CSRs direct them to the hidden checkout?

Reply:
patrons don't really renew themselves, it happens automatically in zuora. At present they get unlimited free tickets due to a missing eventbrite integration as we sell tickets directly there now.  The check is based on the dates in zuora, so it automatically would reset every year anyway.
I don't think we should be acquiring new patrons, certainly not in any significant numbers.